dotnet/src/SemanticKernel/Memory/VolatileMemoryStore.cs

```diff
@@ -36,7 +42,8 @@ public class VolatileMemoryStore<TEmbedding> : VolatileDataStore<IEmbeddingWithM
 
         EmbeddingReadOnlySpan<TEmbedding> embeddingSpan = new(embedding.AsReadOnlySpan());
```

Comment:
@dluc, @shawncal curious about use of new() here. We don't use it anywhere else AFAIK - just want to sync on code style here.

Reply:
Good question, I think we can add the agreement on this to PR review doc. My preference will be to use `var dictionary = new Dictionary<string,string>();` syntax, but I'm open for suggestions.

Reply:
FWIW we are using it a bit in the kernel already -- I like it:
![image](https://user-images.githubusercontent.com/35783676/224798380-239d2c12-d570-4c80-8e05-b16617e76876.png)

Reply:
I far prefer 
`StrongType<Thing> foo = new();`
...over...
`var randomStuff = new StrongType<Thing>();`

I see `new() `as shorthand and `var` as cheating. ;)

More importantly, I like seeing exactly what "foo" is at a glance, rather than how it was made or fetched.